InAppExample/android/app/src/main/java/com/inappexample/MainActivity.kt

```diff
@@ -10,6 +11,9 @@ class MainActivity : ReactActivity() {
      * Returns the name of the main component registered from JavaScript. This is used to schedule
      * rendering of the component.
      */
+    override fun onCreate(savedInstanceState: Bundle?) {
```

Comment:
@JeremyWright Not having this here seems to be creating issues with backgrounding the app